### DISCUSSION TITLE
Application not accepted because of location issues --> I'm an international student on summer break

### DISCUSSION THREAD

**Discussion by belaqua2020** [community/community#56529] [GitHub Education]
### Hi everyone,

Hello,

I go to school at a UC but as an international student, our break began in mid-May and so I am back home outside of the United States/California, but am actively still enrolled with a graduation date of 2024. This proof is provided in my document of validation that I uploaded, however I keep getting the following messages and I am unsure how to proceed:

"The school you selected does not appear to have a campus location in your country. Your school-provided academic affiliation document must indicate virtual learning if you are remote.

You appear not to be near any campus location for the school you have selected. If you are a distance learner then your school-provided academic affiliation documentation must state so."

**Comment by samus-aran** [comment]
Hi @belaqua2020 

For remote student applicants: please ensure that your evidence indicates that you are a remote student. If possible, we encourage students to apply whilst on campus. This will assist us in confirming your student status.

I would also suggest that you ensure your evidence correspond to details in your GitHub profile. The verification process looks at your GitHub username, email and name ( as well as other points) to cross reference the evidence submitted. Do you use a variation of your name or a nickname?  It’s these small but important points to consider when applying that can make a much smoother process.
